homeassistant/components/notify/kodi.py

```diff
@@ -0,0 +1,76 @@
+﻿"""
+Kodi notification service.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/notify.kodi/
+"""
+import logging
+import voluptuous as vol
+
+from homeassistant.const import (ATTR_ICON, CONF_HOST, CONF_NAME,
+                                 CONF_PORT, CONF_USERNAME, CONF_PASSWORD)
+from homeassistant.components.notify import (ATTR_TITLE, ATTR_TITLE_DEFAULT,
+                                             ATTR_DATA, PLATFORM_SCHEMA,
+                                             BaseNotificationService)
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+REQUIREMENTS = ['jsonrpc-requests==0.3']
+
+DEFAULT_NAME = 'Kodi'
+DEFAULT_PORT = 8080
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_HOST): cv.string,
+    vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,

```

Comment:
@chrom3 you are redefining `CONF_NAME` here, it is already defined in `notify/__init__.py`. This is the reason you have unreliable results on execution - dict keys are not ordered so it randomly picks one